airbyte-connector-builder-server/connector_builder/generated/models/connector_manifest.py

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+
+from __future__ import annotations
+from datetime import date, datetime  # noqa: F401
+
+import re  # noqa: F401
+from typing import Any, Dict, List, Optional  # noqa: F401
+
+from pydantic import AnyUrl, BaseModel, EmailStr, validator  # noqa: F401
+from connector_builder.generated.models.check_stream import CheckStream
+from connector_builder.generated.models.declarative_stream import DeclarativeStream
+
+
+class ConnectorManifest(BaseModel):
```

Comment:
FYI @girarda @brianjlai since this PR is having the connector builder server OpenAPI spec reference the connector manifest schema directly, it is now generating a lot more models.

Is this setup okay to move forward with for now? I'm assuming we will either start to consume these models or remove them as part of https://github.com/airbytehq/airbyte/issues/20044